src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTriggerParametersTest.java

```diff
@@ -76,15 +64,12 @@
      */
     @Before
     public void setup() {
-        mockStatic(PluginImpl.class);
-        PluginImpl plugin = mock(PluginImpl.class);
-        when(PluginImpl.getInstance()).thenReturn(plugin);
-
         config = Setup.createConfig();
         GerritServer server = new GerritServer(PluginImpl.DEFAULT_SERVER_NAME);
         server.setConfig(config);
-        when(plugin.getServer(eq(PluginImpl.DEFAULT_SERVER_NAME))).thenReturn(server);
-        when(plugin.getFirstServer()).thenReturn(server);
```

Comment:
Yes, at some point in time it had become some strange abomination with mixed JenkinsRule and PowerMock, when the test didn't pass I started with simplifying the test code to understand what was going wrong in the tests and that fixed the issues.